examples/nitclk/synchronize-tclk.py

```diff
@@ -0,0 +1,167 @@
+# Demonstrates how to configure multiple NI Arbitrary Waveform Generators to generate synchronized waveforms with TClk
+# 
+# The gRPC API is built from the C API. NI-FGEN documentation is installed with the driver at:
+# C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niFgen\documentation\English\SigGenHelp.chm
+#
+# The gRPC API is built from the C API. NI-TClk documentation is installed with the driver at:
+# C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niTClk\documentation\English\nitclk.chm
+#
+# Getting Started:
+#
+# To run this example, install "NI-FGEN Driver" on the server machine.
+# Link : https://www.ni.com/en-us/support/downloads/drivers/download.ni-fgen.html
+#
+# For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC Client" wiki page.
+# Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+#
+# Running from command line:
+#
+# Server machine's IP address and port number can be passed as separate command line arguments.
+#   > python synchronize-tclk.py <server_address> <port_number>
+# If they are not passed in as command line arguments, then by default the server address will be "localhost:31763"
+
+import grpc
+import sys
+import nifgen_pb2 as nifgen_types
+import nifgen_pb2_grpc as grpc_nifgen
+import nitclk_pb2 as nitclk_types
+import nitclk_pb2_grpc as grpc_nitclk
+import session_pb2 as session_types
+import session_pb2_grpc as grpc_session
+import matplotlib.pyplot as plt
+
+server_address = "localhost"
+server_port = "31763"
+
+# Take resource name . This example doesn't work on simulation, so resource name is required.
+resources = input("Enter comma separated resource names (PXI1Slot2,PXI1Slot3): ").split(",")
```

Comment:
take in resource as arguments similar to other tclk example. what would happen if only 1 resource is passed?

Reply:
Difference between this and tclk-scope example is, FGEN can't be simulated. So users will always have to pass in resource name (and hence server address and port number) if we make it a parameter. If we make `resources` as first parameter, then parameter list won't match other examples.

If user inputs only one resource name, `split` will create array of one element.

Reply:
With that, I think the comment and input message here needs to be updated. If Harsha is ok with taking user input, you should change above lines to

```
# Read FGEN resources to be configured
 resources = input("Enter comma separated FGEN resource names (PXI1Slot2, PXI1Slot3): ").split(",").strip()
```

The `strip` call would remove any leading/trailing whitespaces.

Reply:
changed the input part

Reply:
changed this